server/config.go

```diff
@@ -22,6 +22,7 @@ type Config struct {
 	Network        *network.Config
 	DataDir        string
 	Seal           bool
+	Tracker        bool
```

Comment:
What do you think about renaming this to `TrackerEnabled`, since you only store the value indicating if the tracker should be initialized in the first place?

Reply:
Actually, great catch - we decided to remove the tracker from the server entirely. (@Kourin1996 )
[Removed](https://github.com/0xPolygon/polygon-edge/pull/386/commits/099bb19976e1301cba0bb62c68785e6ce8149be2)